src/main/resources/templates/fragments/header.html

```diff
@@ -29,7 +29,7 @@
                     </div>
 
                      <div class="govuk-header__content">
-                        <a id="proposition-name" href="/" class="govuk-header__link govuk-header__link--service-name"> File company accounts </a>
+                        <a id="proposition-name" href="/" class="govuk-header__link govuk-header__link--service-name"> File your CIC report and accounts </a>
```

Comment:
Should we be adding CIC details yet since the application won't support CICs until much later?

Reply:
Just going by the prototype

Reply:
just going by the prototype given in the story https://ch-accounts-prototype-dev.herokuapp.com/small-full/small-full-balance-sheet-mvp